assets/javascripts/templates/pages/root_tmpl.coffee.erb

```diff
@@ -39,7 +39,7 @@ app.templates.intro = """
       <li>DevDocs works <a href="/offline">offline</a>, on mobile, and can be installed on <a href="https://chrome.google.com/webstore/detail/devdocs/mnfehgbmkapmjnhcnbodoamcioleeooe">Chrome</a>.
       <li>For the latest news, follow <a href="https://twitter.com/DevDocs">@DevDocs</a>.
       <li>DevDocs is free and <a href="https://github.com/freeCodeCamp/devdocs">open source</a>.
-          <iframe class="_github-btn" src="//ghbtns.com/github-btn.html?user=freeCodeCamp&repo=devdocs&type=watch&count=true" allowtransparency="true" frameborder="0" scrolling="0" width="100" height="20"></iframe>
```

Comment:
Another idea, not sure if you tried it, would be to add the sandbox attribute.

Reply:
IMO a static SVG is better anyway, so there’s no point in using an iframe.